plugins/links.rb

```diff
@@ -26,7 +26,7 @@ def initialize
   end
 
   def on_message(message, matches)
-    url = matches[0]
+    url = matches[0].split('|')[0]
```

Comment:
This is the hot-patch on my instance of Linkbot to fix Slack links.

I'm not proud. It's this or edit the plugin's regex to not collect pipe-and-what-comes-after which is intimidating.